static/success-banner.css

```diff
@@ -1 +1 @@
-body{display:flex;justify-content:center;background-color:#535760;font-family:'Helvetica Thin', Helvetica, sans-serif;color:#ffffff}.horizontal,.vertical{display:flex}.vertical{flex-direction:column}.center{display:flex;justify-content:center;align-items:center}.button{display:flex;align-items:center;justify-content:center;width:165px;padding:14px 44px;margin:7.5px;background-color:#313339;color:#fff;font-weight:bold;text-decoration:none;border-radius:3px}.button>img{margin:0 10px}.icon{max-height:24px}.icon.github{margin-right:15px}.icon.twitter{margin-right:8px;margin-left:-4px}footer{margin-top:30px;display:flex;align-items:center;justify-content:center}footer>img{margin:0 0.5em}footer>.brand{max-height:24px}h1{display:inline-block;padding:5px;margin-top:0;font-weight:normal;font-size:26px}.etcher-logo{margin-left:8px;width:130px}
+body{display:flex;justify-content:center;background-color:#535760;font-family:'Helvetica Thin', Helvetica, sans-serif;color:#ffffff}.horizontal,.vertical{display:flex}.vertical{flex-direction:column}.center{display:flex;justify-content:center;align-items:center}.button{display:flex;align-items:center;justify-content:center;width:165px;padding:14px 44px;margin:7.5px;background-color:#313339;color:#fff;font-weight:bold;text-decoration:none;border-radius:3px}.button>img{margin:0 10px}.icon{max-height:24px}.icon.github{margin-right:15px}.icon.twitter{margin-right:8px;margin-left:-4px}footer{margin-top:30px;display:flex;align-items:center;justify-content:center}footer>img,footer>a{margin:0 0.5em}footer>a>.brand{max-height:24px}h1{display:inline-block;padding:5px;margin-top:0;font-weight:normal;font-size:26px}.etcher-logo{margin-left:8px;width:130px}
```

Comment:
I believe he's away on vacation now so I've opened this as a separate issue https://github.com/resin-io/etcher-homepage/issues/76 we can fix later.